overrides/kubejs/server_scripts/ColdSweat.js

```diff
@@ -10,4 +10,93 @@ ServerEvents.recipes(event => {
         B: 'minecraft:bricks',
         C: 'minecraft:campfire'
     })
+    event.remove({ id: 'cold_sweat:thermometer'})
```

Comment:
Good recipe, only change i'd request is that we use bronze or just gold.
Brass requires zinc which puts brass-anything post-nether (Which is after steel by a good bit)